CHANGES.md

```diff
@@ -40,6 +40,15 @@ OpenSSL 3.0
 
    *Boris Pismenny, John Baldwin and Andrew Gallatin*
 
+ * The signature of the `copy` functional parameter of the
```

Comment:
I'd suggest _function_ instead of _functional_ here and in the POD files.

Reply:
I meant by this that the parameter is a function instead of it is just another parameter of the EVP_PKEY_meth_set_copy() function. Could you please suggest better wording as `signature of ... function parameter of the ... function` sounds a little bit silly to me.